tests/misc/fork.c3

```diff
@@ -14,6 +14,7 @@ function main : () -> int = {
     return 0;
 }
 
+/// @FOR x86_64_linux
```

Comment:
Is this just temporary until we've implemented a `clone3` based `fork` for arm?

Reply:
Yes, although that's not really a priority